### ISSUE TITLE
toy demo

### ISSUE THREAD

**Issue by mas-dse-greina** [jakeret/tf_unet#120]
I've been trying to run the toy demo python notebook. The code runs, but it doesn't seem to converge. I keep getting the same verification error of around 17% for every epoch and when it finishes it predicts all black images (i.e. all 0s).

Has anyone else run into this? I'm using Keras 2 with TF 1.3 in a Python 3 Anaconda distro.

Thanks

**Comment by mas-dse-greina** [comment]
Ok. It seems to run fine on a different machine. Maybe it just needs to be trained for more epochs to show better convergence?? Or different learning rate/momentum parameters?

**Comment by jakeret** [comment]
I'm not setting any seeds or something. Might be that in some cases the training gets stuck. So far I never encountered anything thelike

**Comment by mas-dse-greina** [comment]
Yes, I tried to increase the momentum which seemed to help so it is probably just getting stuck during SGD.
